resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -6,33 +6,47 @@
 from lxml import etree
```

Comment:
Wrong import order. Please, follow [PEP8 rules](https://www.python.org/dev/peps/pep-0008/#imports).

Also, why are you skipping invalid names and import error for the **whole** file? (line 2)